catalog/spatial/registry/registry-api-impl/src/main/java/org/codice/ddf/registry/api/impl/RegistryStorePublisher.java

```diff
@@ -210,7 +210,7 @@ public void handleEvent(Event event) {
                 (RegistryStore) bundleContext.getService(((ServiceEvent) event.getProperty(
                         EventConstants.EVENT)).getServiceReference());
 
-        if (!previousAutoPush && registryStore.isAutoPush()) {
+        if (registryStore.isAutoPush()) {
```

Comment:
With the change to this conditional statement, you should be able to remove ` && !registryStore.isAutoPush()` from the else statement.